wikimedia/wikipedia-ios
# Native editor icons update

**Phabricator:** https://phabricator.wikimedia.org/T344508

### Notes
* This PR updates the icons used on the new native editor to SFSymbols
* Icons used elsewhere will be replaced with the future icons work
* Multicolor icons also will be added in a future PR, as more changes will need to be done to the base components 

### Test Steps
1. Run the app on the Staging target
2. Check the new icons against the Figma file
